source/ConversationPanel.cpp

```diff
@@ -77,6 +78,15 @@ ConversationPanel::ConversationPanel(PlayerInfo &player, const Conversation &con
 	else if(player.Flagship())
 		subs["<ship>"] = player.Flagship()->Name();
 
+	Date date = player.GetDate();
+	subs["<current year>"] = date.CurrentYear();
```

Comment:
```suggestion
	subs["<current year>"] = to_string(date.Year());
```
All this function does is taking the output of a single public Date member function and processing it with to_string. This doesn't need to be a separate function IMO.